frontend/src/components/modalLeftWindow.jsx

```diff
@@ -0,0 +1,11 @@
+import "../css/personalPage.css"
+
+function ModalLeftWindow({body}) {
```

Comment:
This isn't really a modal window...it doesn't pop up and go on top of all the other page content...it's more like a panel

Reply:
Changed to LeftPanel